src/babel/api/register/node.js

```diff
@@ -37,6 +38,12 @@ var only;
 var oldHandlers   = {};
 var maps          = {};
 
+var cwd = process.cwd();
```

Comment:
A check of `require.main ? require.main.filename : process.cwd()` would be more reliable since the cwd may not necessarily be the directory of the entry file.